src/Microsoft.Identity.Client/PlatformsCommon/Factories/PlatformProxyFactory.cs

```diff
@@ -41,20 +42,22 @@ internal static class PlatformProxyFactory
         /// </summary>
         public static IPlatformProxy CreatePlatformProxy(ICoreLogger logger)
         {
+            var finalLogger = logger ?? MsalLogger.Create(Guid.NewGuid(), null, false);
```

Comment:
>MsalLogger.Create(Guid.NewGuid() [](start = 40, length = 32)

Would it make sense to create a lazy loaded internal object which is the EmptyLogger ? If this is the only place it probably doesn't make a lot of sense, but then it might still be a bit more explicit and it may be re-used across the lifetime of the object/process (a process level static)?

Reply:
Good idea.  done.

---
In reply to: [251651195](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/844#discussion_r251651195) [](ancestors = 251651195)